src/bt_peer_connection.cpp

```diff
@@ -358,9 +358,12 @@ namespace libtorrent {
 		stats_counters().inc_stats_counter(counters::num_outgoing_dht_port);
 	}
 
-	void bt_peer_connection::send_piece_message(message_type const type
```

Comment:
these are the function definitions. the ``const`` *do* have meaning here. would you mind putting them back?